lib/src/signed_video_openssl.c

```diff
@@ -319,51 +320,80 @@ openssl_hash_data(void *handle, const uint8_t *data, size_t data_size, uint8_t *
 
 /* Initializes EVP_MD_CTX in |handle| with |hash_algo.type|. */
 svrc_t
-openssl_init_hash(void *handle)
+openssl_init_hash(void *handle, bool fallback_to_gop_level)
```

Comment:
The name `fallback_to_gop_level` id very misleading and narrow. What you do is selecting between two hash contexts, either `primary_ctx` or `secondary_ctx`. So I would flip the interpretation and name the input `bool primary`, leaving out the "ctx" part since the user of the crypto handle does not know the naming of the member.